public/assets/images/icons/project/Conservation.tsx

```diff
@@ -2,7 +2,12 @@ import React from 'react';
 
 const Conservation = () => {
   return (
-    <svg viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <svg
+      viewBox="0 0 32 32"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      style={{ marginBottom: '0.3em' }}
+    >
```

Comment:
This is because defining a style in the common container for all icons :  `projectEcosystemOrTypeContainer` is not working for this icon specifically due to difference in it's positioning